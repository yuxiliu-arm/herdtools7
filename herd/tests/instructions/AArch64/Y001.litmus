AArch64 Y001
(* MADD *)

{
    0: X1 = 2;
    0: X2 = 3;
    0: X3 = 5;
}

P0 ;
MADD X4, X1, X2, X3;

forall (
    0: X1 = 2 /\
    0: X2 = 3 /\
    0: X3 = 5 /\
    0: X4 = 11
)
