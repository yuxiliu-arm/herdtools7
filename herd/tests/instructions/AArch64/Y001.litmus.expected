Test Y001 Required
States 1
0:X1=2; 0:X2=3; 0:X3=5; 0:X4=11;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=2 /\ 0:X2=3 /\ 0:X3=5 /\ 0:X4=11)
Observation Y001 Always 1 0
Hash=e6b2ade0b91d469a9aac23a9d4b3e3d4

